addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -177,7 +177,7 @@ Resources:
               StringEquals:
                 sts:ExternalId: !Ref ExternalId
       ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/AWSServiceCatalogAdminFullAccess
```

Comment:
Looks like those changes should be part of #9

Reply:
Done